wiki-service/src/main/java/org/exoplatform/wiki/service/rest/NotesRestService.java

```diff
@@ -78,10 +78,13 @@ public NotesRestService(WikiService noteBookService, ResourceBundleService resou
   @Path("/note/{noteBookType}/{noteBookOwner:.+}/{noteId}")
   @Produces(MediaType.APPLICATION_JSON)
   @RolesAllowed("users")
-  public Response getNote(@Context UriInfo uriInfo,
-                          @PathParam("noteBookType") String noteBookType,
-                          @PathParam("noteBookOwner") String noteBookOwner,
-                          @PathParam("noteId") String noteId) {
+  @ApiOperation(value = "Get note by notes params", httpMethod = "GET", response = Response.class, notes = "This get the not if the authenticated user has permissions to view the objects linked to this note.")
+  @ApiResponses(value = { @ApiResponse(code = 200, message = "Request fulfilled"),
+          @ApiResponse(code = 400, message = "Invalid query input"), @ApiResponse(code = 403, message = "Unauthorized operation"),
+          @ApiResponse(code = 404, message = "Resource not found") })
+  public Response getNote(@ApiParam(value = "NoteBook Type", required = true) @PathParam("noteBookType") String noteBookType,
+                          @ApiParam(value = "NoteBook Owner", required = true) @PathParam("noteBookOwner") String noteBookOwner,
+                          @ApiParam(value = "Note id", required = true) @PathParam("noteId") String noteId) {
     try {
       Page note = noteBookService.getPageOfWikiByName(noteBookType, noteBookOwner, noteId);
```

Comment:
no permission check? how do we ensure only authorised readers will be able to load a page?